cfy_manager/components/manager_status_reporter/__init__.py

```diff
@@ -0,0 +1,14 @@
+#########
```

Comment:
there isn't really a reason to make a directory, could just do `cfy_manager/components/manager_status_reporter.py`

Reply:
ditto for the other 2

Reply:
Well there are a couple of the same small component that are in there on folder like sanity, patch, ip setter and cli

Reply:
yea those would ideally be changed too :P